src/CalculateEnergy.cpp

```diff
@@ -594,7 +594,30 @@ bool CalculateEnergy::MoleculeInter(Intermolecular &inter_LJ,
     GOMC_EVENT_START(1, GomcProfileEvent::EN_MOL_INTER);
     uint length = mols.GetKind(molIndex).NumAtoms();
     uint start = mols.MolStart(molIndex);
+#ifdef GOMC_CUDA
+    std::vector<int> cellVector, cellStartIndex, mapParticleToCell;
+    std::vector< std::vector<int> > neighborList;
+    cellList.GetCellListNeighbor(box, currentCoords.Count(),
+                                cellVector, cellStartIndex, mapParticleToCell);
+    neighborList = cellList.GetNeighborList(box);
+    
+    XYZArray cCoords(length);
+    std::vector<double> MolCharge;
+    for(uint p = 0; p < length; p++) {
+      cCoords.Set(p, currentCoords[start + p]);
+      //MolCharge.push_back(particleCharge[start + p] * GetLambdaCoef(molIndex, box););
+    }
+    // Should be different in production
+    XYZArray molCoords = cCoords;
```

Comment:
@GregorySchwing why are you overriding the molCoords? This variable supposed to hold the new molecule coordinate.